ts/models/conversation.ts

```diff
@@ -726,6 +734,13 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
         lokiProfile: UserUtils.getOurProfile(),
       };
 
+      const updateApprovalNeeded =
+        !this.isApproved() && (this.isPrivate() || this.isMediumGroup() || this.isClosedGroup());
+      if (updateApprovalNeeded) {
+        await this.setIsApproved(true);
+        await forceSyncConfigurationNowIfNeeded();
```

Comment:
make this `await forceSyncConfigurationNowIfNeeded` void, we are sending a message and shouldn't delay too much the effective sending